app/domain_models/follow_status_data.rb

```diff
@@ -0,0 +1,83 @@
+class FollowStatusData
```

Comment:
Usually, we do not memo the code if the code has been clear enough.
And, you should make it as clear as possible by code rather than using memo.